src/Tools/MicroBuild/Build.proj

```diff
@@ -25,6 +25,8 @@
 
     <MSBuild Projects="$(ProjectDir)Roslyn.sln" BuildInParallel="true" Properties="DeployExtension=false" />
 
+    <Exec Command="$(NuGetPackageRoot)\GitLink\$(GitLinkVersion)\lib\net45\GitLink.exe $(ProjectDir) -u https://github.com/dotnet/roslyn -f Roslyn.sln -c $(Configuration)" />
```

Comment:
About performance: @AArnott is helping us working on a pdb-only version. This should no longer require the tool to load the solution making it much faster. This will be v3+.